minimega: vnc crashes on blank lines

Thought this was fixed by #758 but apparently it's still an issue:

```
panic: runtime error: index out of range

goroutine 207 [running]:
panic(0xce72c0, 0xc4200100e0)
        /usr/lib/golang/src/runtime/panic.go:481 +0x3e6
main.(*vncKBPlayback).playFile(0xc4215b4500)
        minimega/src/minimega/vnc_playback.go:206 +0xd61
created by main.(*vncKBPlayback).Play
        minimega/src/minimega/vnc_playback.go:343 +0x29c
```

File:

```
$ cat test.vnc
503553298:KeyEvent,true,h
54053754:KeyEvent,false,h

```

This appeared in the logs:

```
2016/11/28 18:10:45 ERROR vnc_record.go:224: strconv.ParseInt: parsing "": invalid syntax
```

--- a/src/minimega/vnc_playback.go
+++ b/src/minimega/vnc_playback.go
@@ -192,8 +192,9 @@ fileLoop:
 				// Parse the event
 				s := strings.SplitN(pr.scanner.Text(), ":", 2)
 
-				// Skip blank lines
-				if len(s) == 0 {
+				// Skip malformed commands and blank lines
+				if len(s) != 2 {
+					log.Debug("malformed vnc command: %s", pr.scanner.Text())
 					continue
 				}
 

--- a/src/minimega/vnc_record.go
+++ b/src/minimega/vnc_record.go
@@ -215,10 +215,17 @@ func getDuration(filename string) time.Duration {
 	scanner := bufio.NewScanner(f)
 	for scanner.Scan() {
 		s := strings.SplitN(scanner.Text(), ":", 2)
+		// Ignore blank and malformed lines
+		if len(s) != 2 {
+			log.Debug("malformed vnc statement: %s", scanner.Text())
+			continue
+		}
+
 		// Ignore comments in the vnc file
 		if s[0] == "#" {
 			continue
 		}
+
 		i, err := strconv.Atoi(s[0])
 		if err != nil {
 			log.Errorln(err)

